models/urbanczik_synapse.h

```diff
@@ -40,51 +40,66 @@
 namespace nest
 {
 
-/** @BeginDocumentation
-@ingroup Synapses
-@ingroup stdp
+/* BeginUserDocs: synapse, spike-timing-dependent plasticity
 
-Name: urbanczik_synapse - Synapse type for a plastic synapse after Urbanczik and Senn.
+Short description
++++++++++++++++++
 
-Description:
+Synapse type for a plastic synapse after Urbanczik and Senn
 
-urbanczik_synapse is a connector to create Urbanczik synapses as defined in [1] that can connect suitable
-multicompartment models. In contrast to usual STDP, the change of the synaptic weight does not only depend on the pre-
-and postsynaptic spike timing but also on the postsynaptic dendritic potential.
+Description
++++++++++++
 
-Urbanczik synapses require archiving of continuous quantities. Therefore they can only be connected to neuron models
-that are capable of doing this archiving. So far, the only compatible model is pp_cond_exp_mc_urbanczik.
+urbanczik_synapse is a connector to create Urbanczik synapses as defined in
+[1]_ that can connect suitable :ref:`multicompartment models
+<multicompartment-models>`. In contrast to usual STDP, the change of the
+synaptic weight does not only depend on the pre- and postsynaptic spike timing
+but also on the postsynaptic dendritic potential.
 
-Parameters:
+Urbanczik synapses require archiving of continuous quantities. Therefore they
```

Comment:
Is there a better word than quantity?

Reply:
@jessica-mitchell Yes. To be more precise one could write: Urbanczik synapses require the archiving of the dendritic membrane potential which is continuous in time. Therefore ..."

Reply:
that sounds better

Reply:
@jessica-mitchell Okay, then I will add that.

Reply:
Could this be reworded as suggested?